backend/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16-alpine3.17 as develop
+FROM node:20.6.1-alpine3.18 as develop
```

Comment:
vi har flera filer sen att ändra nodeversion i. sen borde vi kanske lägga in en `.nvmrc` fil som specar vilken version vi kör